packages/proxmark3-git/build.sh

```diff
@@ -0,0 +1,25 @@
+TERMUX_PKG_HOMEPAGE="https://github.com/RfidResearchGroup/proxmark3"
+TERMUX_PKG_DESCRIPTION="The Swiss Army Knife of RFID Research - RRG/Iceman repo"
+TERMUX_PKG_LICENSE="GPL-2.0"
+TERMUX_PKG_MAINTAINER="Marlin Sööse <marlin.soose@laro.se>"
+TERMUX_PKG_REVISION=2
+TERMUX_PKG_VERSION="2021.05.11-a6712295"
+TERMUX_PKG_SRCURL="https://github.com/RfidResearchGroup/proxmark3/archive/a6712295c7ad9b78962be1d21a8eb2fe0688093d.tar.gz"
+TERMUX_PKG_SHA256="02eb06d82dddae1caad1d73e727250a7455d8f30950feb5dd9b6b391a460dc73"
+TERMUX_PKG_BUILD_DEPENDS="readline,libbz2"
+TERMUX_PKG_BUILD_IN_SRC="true"
+TERMUX_PKG_BLACKLISTED_ARCHES="i686, x86_64"
+
+termux_step_post_configure() {
+        LDLIBS="-L${TERMUX_PREFIX}/lib" INCLUDES_CLIENT="-I${TERMUX_PREFIX}/include" CFLAGS="-I${TERMUX_PREFIX}/include"
+}
+
+termux_step_make() {
+        TERMUX_PKG_EXTRA_MAKE_ARGS="client -j $TERMUX_MAKE_PROCESSES CC=$CC CXX=$CXX LD=$CXX cpu_arch=generic SKIPREVENGTEST=1 SKIPQT=1 SKIPBT=1 SKIPPTHREAD=1 SKIPGPU=1 PLATFORM=PM3GENERIC"
```

Comment:
You can set this under `TERMUX_PKG_BLACKLISTED_ARCHES`, and then not override termux_step_make

Reply:
Seems like it would really confuse things, the args aren't architectures.

Why would this be desired?

Reply:
I meant having:

```
TERMUX_PKG_BUILD_IN_SRC="true"
TERMUX_PKG_BLACKLISTED_ARCHES="i686, x86_64"
TERMUX_PKG_EXTRA_MAKE_ARGS="client -j $TERMUX_MAKE_PROCESSES CC=$CC CXX=$CXX LD=$CXX cpu_arch=generic SKIPREVENGTEST=1 SKIPQT=1 SKIPBT=1 SKIPPTHREAD=1 SKIPGPU=1 PLATFORM=PM3GENERIC"
```

But yes, that won't work (maybe you noticed) because CC and friends are not defined when the build.sh is sourced.

I would prefer adding it to `termux_step_post_configure()` as well, so that we have

```
termux_step_post_configure() {
        LDLIBS="-L${TERMUX_PREFIX}/lib" INCLUDES_CLIENT="-I${TERMUX_PREFIX}/include" CFLAGS="-I${TERMUX_PREFIX}/include"
        TERMUX_PKG_EXTRA_MAKE_ARGS="client -j $TERMUX_MAKE_PROCESSES CC=$CC CXX=$CXX LD=$CXX cpu_arch=generic SKIPREVENGTEST=1 SKIPQT=1 SKIPBT=1 SKIPPTHREAD=1 SKIPGPU=1 PLATFORM=PM3GENERIC"
}
```

That way we do not need to override termux_step_make, which should be avoided if possible

Reply:
Ah, I understand. Thanks for explaining.